sys/net/gnrc/network_layer/sixlowpan/frag/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+rsource "fb/Kconfig"
+rsource "rb/Kconfig"
+rsource "vrb/Kconfig"
```

Comment:
These should depend on `MODULE_GNRC_IPV6_SIXLOWPAN_FRAG`:
```Kconfig
if MODULE_GNRC_IPV6_SIXLOWPAN_FRAG

rsource "fb/Kconfig"
rsource "rb/Kconfig"
rsource "vrb/Kconfig"

endif # MODULE_GNRC_IPV6_SIXLOWPAN_FRAG
```
(Note that this is not a 'conditional include though, it adds this dependency to all the symbols inside).